manifests/init.pp

```diff
@@ -53,8 +57,10 @@
 #
 class hash_resources(
   $resources = {},
+  $realize = {},
```

Comment:
Would this be better as an array? I've never used `realize`, but seems like it only takes an array and you end up converting this anyway.